From 91751a7f295b047c40cdc9377b36b811ef1a0181 Mon Sep 17 00:00:00 2001
From: Sven van Haastregt <sven.vanhaastregt@arm.com>
Date: Tue, 28 Jun 2022 11:28:52 +0100
Subject: [PATCH 059/172] [CI] Exclude some files from code style checks

Some files cannot be parsed on their own as they for example require a
macro to be defined.  To avoid spurious failures of clang-tidy,
exclude such files from the code style checks.

Remove spirv.hpp from the list as it is no longer in the tree.

(cherry picked from commit 09749079fb73f3b0e799adf7955dfbb8bb63afb4)
---
 .github/workflows/check-code-style.yml | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/.github/workflows/check-code-style.yml b/.github/workflows/check-code-style.yml
index 087a4088..b4521564 100644
--- a/.github/workflows/check-code-style.yml
+++ b/.github/workflows/check-code-style.yml
@@ -43,7 +43,12 @@ jobs:
     - name: Gather list of changes
       id: gather-list-of-changes
       run: |
-        git diff -U0 --no-color ${{ github.sha }}^ -- include lib ':(exclude)lib/SPIRV/libSPIRV/spirv.hpp' > diff-to-inspect.txt
+        git diff -U0 --no-color ${{ github.sha }}^ -- include lib \
+            ':(exclude)include/LLVMSPIRVExtensions.inc' \
+            ':(exclude)lib/SPIRV/libSPIRV/SPIRVErrorEnum.h' \
+            ':(exclude)lib/SPIRV/libSPIRV/SPIRVOpCodeEnum.h' \
+            ':(exclude)lib/SPIRV/libSPIRV/SPIRVOpCodeEnumInternal.h' \
+            > diff-to-inspect.txt
         if [ -s diff-to-inspect.txt ]; then
           # Here we set an output of our step, which is used later to either
           # perform or skip further steps, i.e. there is no sense to install
-- 
2.20.1

